README.md

```diff
@@ -0,0 +1,3 @@
+# blimflix
```

Comment:
I think we should remove these, PR should only contain code related to the feature you worked on.